api/cache/projects.js

```diff
@@ -174,25 +174,31 @@ async function getInitialTriageProjectsFromJira(startAt, itArray) {
 
   const itTotalData = parseInt(jiraIt.data.total);
 
-  jiraIt.data.issues.map((x) =>
-    itArray.push({
+  await Promise.all(jiraIt.data.issues.map(async (x) => {
+    const project = {
       it_key: x['key'],
       name: x['fields'].summary,
       description: x['fields'].description,
       client: x['fields'].customfield_10027,
       video: x['fields'].customfield_10159 ?? '',
       scope: x['fields'].customfield_10090,
-      sector: x['fields'].customfield_10148?.value ?? ''
-    }),
-  );
+      sector: x['fields'].customfield_10148?.value ?? '',
+    }
+
+    const videoFile =  await videoHelper.getVideoFileFromVimeo(project.video)
+    project.video_file = videoFile
```

Comment:
Just need a semi-colon at the end of this line, for consistency with the style of the rest of the API code